openfga/v1/openfga_service.proto

```diff
@@ -202,13 +203,18 @@ service OpenFGAService {
       tags: ["Relationship Queries"]
       operation_id: "Check"
       description:
-        "The Check API queries to check if the user has a certain relationship with an object in a certain store.\n"
+        "The Check API returns whether a given user has a relationship with a given object in a given store.\n"
+        "The `user` field of the request can be a specific target, such as `user:anne`, or a userset (set of users) such as `user:*` or `group:marketing#member`.\n"
+        "To arrive at a result, the API uses: an authorization model, explicit tuples written through the Write API, and implicit tuples that exist by virtue of applying set theory.\n"
         "A `contextual_tuples` object may also be included in the body of the request. This object contains one field `tuple_keys`, which is an array of tuple keys. Each of these tuples may have an associated `condition`.\n"
         "You may also provide an `authorization_model_id` in the body. This will be used to assert that the input `tuple_key` is valid for the model specified. "
         "If not specified, the assertion will be made against the latest authorization model ID. It is strongly recommended to specify authorization model id for better performance.\n"
         "You may also provide a `context` object that will be used to evaluate the conditioned tuples in the system. It is strongly recommended to provide a value for all the input parameters of all the conditions, to ensure that all tuples be evaluated correctly.\n"
         "The response will return whether the relationship exists in the field `allowed`.\n\n"
-        "## Example\n"
+        "In general, if a Check API responds with `{allowed: true}`, then you can expect the equivalent ListObjects query to return the object, and viceversa. \n"
```

Comment:
Should we note that some exceptions apply?

e.g. ListObjects only returns tuples that it can compute within a time-frame?